src/apollo/modules/map/pnc_map/CMakeLists.txt

```diff
@@ -13,7 +13,8 @@ add_library(map_route_segments route_segments.cc)
 target_link_libraries(map_route_segments common::proto
                                        map::hdmap
                                        map::pnc_map_path
-                                       routing::routing_gflags)
+                                       routing::routing_gflags
+                                       routing::proto)
```

Comment:
Why need this?  The original build file does not list `routing::proto` as a dependence.

Reply:
Update it now. 
The real loss is `map_pnc_map_path` lib which loss `routing::proto` lib depend.
see BUILD file [here](https://github.com/yuzhangbit/apollo_standalone/blob/a7f639636fb663cfa1551f481ad57df8994662d6/src/apollo/modules/map/pnc_map/BUILD#L48)